jack-core/src/com/rapleaf/jack/transaction/DbMetricsImpl.java

```diff
@@ -0,0 +1,157 @@
+package com.rapleaf.jack.transaction;
+
+import com.google.common.base.Stopwatch;
+import org.apache.commons.pool2.impl.GenericObjectPool;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DbMetricsImpl implements DbMetrics {
+
+
+  private Stopwatch lifeTimeStopwatch;
+  private long lastUpdateTime;
+  private int currentConnectionsNumber;
+    /*
+        Raw metrics
+     */
+
+  private long totalConnections;
+  private long maxActiveConnectionsTime;
+  private long minIdleConnectionsTime;
+
+
+  long maxConnectionWaitTime;
+  private long totalConnectionWaitTime;
+  private long totalIdleTime;
+  private long totalActiveTime;
+  private long openedConnectionsNumber;
```

Comment:
`totalConnections` and `openedConnectionsNumber` are kind of confusing. Can you rename `totalConnections` to something more clear?